MakingCoffee/src/MakingCoffeeEx1.java

```diff
@@ -0,0 +1,43 @@
+
+import java.util.Scanner;
+public class MakingCoffeeEx1 {
+
+           public static void main(String[] args) {
+            int waterPerCup = 200;
+            int milkPerCup = 50;
+            int coffeeBeansPerCup = 15;
+            int cupsOfCoffee1 = 2;
```

Comment:
You don't need `cupsOfCoffee1` & `cupsOfCoffee2` variables, your code should  calculate all possible number of cups for appropriate resources.